### DISCUSSION TITLE
How do I get the name of an organization that has not been active since 2011?

### DISCUSSION THREAD

**Discussion by muffinpowered** [community/community#147255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I came up with a nice new name for my organization, but it turns out it's already taken. While looking at the activity of the person who took my organization's name, I noticed that his account has been inactive since 2011. Is it possible to rewrite his name to my organization?

**Comment by davevad93** [comment]
Hi @shizumico ,

This is what the [GitHub Username Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy) states:

> Keep in mind that not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.
>
> If the username you want has already been claimed, consider other names or unique variations. Using a number, hyphen, or an alternative spelling might help you identify a desirable username that's still available.

So, unless there's a violation of the [Trademark Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy#trademark-policy), I don't think it's possible to claim an already used organization name because, as stated above, "**accounts with no visible activity may be in active use**".
